front/components/assistant_builder/DataSourceSelectionSection.tsx

```diff
@@ -95,14 +96,14 @@ export default function DataSourceSelectionSection({
                 FolderIcon
               );
 
+              // Folders are always displayed as leaf items, except for table items.
+              const isLeafItem =
```

Comment:
@Fraggle to not conflict with your method `canBeExanded` I'll live it like this.

Reply:
It will conflict since i updated line 101 too :)
But it's okay, will manage :)